reviews/templates/reviews/review_detail_sidebar.html

```diff
@@ -36,6 +36,14 @@ <h3>
                 </a>
             </li>
         {% endif %}
+        <li>
+            {% trans "Aanvrager(s)" %}: {{ review.proposal.created_by}}
```

Comment:
This defaults to the username, which on the server would be Solis-ID. Not very useful as many of those are just numbers. Preferably use `User.get_full_name`, optionally with email behind it (in brackets) if that's not too long.

Reply:
Check: het ziet er nu zo uit, met het emailadres op een newline in een nested list. 

![Screenshot 2023-03-07 at 16 23 58](https://user-images.githubusercontent.com/31687030/223467068-5a190e18-0dec-416a-9eb0-a7e9385ff1d0.png)

Reply:
Beautiful!